core/parameters/src/config_store.rs

```diff
@@ -330,7 +330,7 @@ mod tests {
     #[cfg(not(feature = "calimero_zero_storage"))]
     fn test_override_runtime_config() {
```

Comment:
Delete this test

Reply:
(aside) @nagisa, could you please confirm if it's fine to delete this test? If I understand the blames correctly, this is from the time when we were shifting runtime configs to config files?

Reply:
I mean fundamentally parameter merging/override functionality still exists & it will continue to exist in the future – we are going to be supporting at least two protocol versions at a time and it would be ideal that the merging of parameter files continues to be tested.

It doesn't need to continue using protocol versions for the test, though. I think a better test here would be one that properly set up some mock test parameter files and merged here and now (rather than piggy-backing on production parameter files, which we may consider simplifying in the future.)